Winleafs.Models/Models/UserSettings.cs

```diff
@@ -390,6 +411,15 @@ private static JToken Migration_6_7(JToken jToken)
 
             return jToken;
         }
+
+
+        [Migration("7", "8")]
+        private static JToken Migration_7_8(JToken jToken)
+        {
+            jToken[nameof(Schedules)] = new JArray();
```

Comment:
Do keep in mind the bug that we had before, this may occur again

Reply:
Yes, noted.